internal/commands/push.go

```diff
@@ -107,45 +105,49 @@ func NewPushCommand(ctx context.Context, logger *log.Logger) *cobra.Command {
 }
 
 func pushBundle(ctx context.Context, repository string, path string) (*ocispec.Descriptor, error) {
-	cli, err := auth.NewClient()
+	memoryStore := content.NewMemory()
+	var err error
+	memoryStore, _, err = buildLayers(ctx, memoryStore, path)
 	if err != nil {
-		return nil, fmt.Errorf("get auth client: %w", err)
-	}
-
-	resolver, err := cli.Resolver(ctx, http.DefaultClient, false)
-	if err != nil {
-		return nil, fmt.Errorf("docker resolver: %w", err)
+		return nil, fmt.Errorf("building layers: %w", err)
 	}
+	allowedMediaTypes := []string{openPolicyAgentConfigMediaType}
+	copyOpts := []oras.CopyOpt{oras.WithAllowedMediaTypes(allowedMediaTypes)}
 
-	memoryStore := content.NewMemoryStore()
-	layers, err := buildLayers(ctx, memoryStore, path)
+	registry, err := content.NewRegistry(content.RegistryOptions{PlainHTTP: false})
```

Comment:
Does specifying `PlainHTTP: false` here do anything? I suspect not as booleans in Go default to `false`.